cns/ipampool/metrics.go

```diff
@@ -117,7 +125,7 @@ func init() {
 		ipamPrimaryIPCount,
 		ipamRequestedIPConfigCount,
 		ipamTotalIPCount,
-	)
+		ipamSubnetExhaustionState)
```

Comment:
the parenthesis needs to be on a newline

Reply:
Fixed